packages/web-runtime/src/store/spaces.ts

```diff
@@ -69,6 +74,17 @@ const actions = {
 
     const spaces = graphResponse.data.value.map((space) => buildSpace(space))
     context.commit('LOAD_SPACES', spaces)
+  },
+  async loadSpaceQuotas(context, { httpAuthenticatedClient }) {
+    // FIXME: Use graphClient after added to libre-graph-api specs https://github.com/owncloud/libre-graph-api
```

Comment:
Why not do that first and wait for that? Is this PR that urgent?

Reply:
I don’t think that it is planned right now.
Maybe @fschade knows more

Reply:
Is it available in the backend side graph api and just not in the spec? Because that would be a low hanging fruit. Or is the endpoint not implemented for graph API at all?

Reply:
It’s not in the specs but implemented in the graph api on the experimental drive

Reply:
@kobergj does it make sense to add it to the graphapi? if yes, Is it a low hanging fruit?

Reply:
@fschade see discussion on other comment, unfortunately it is not that easy...

Reply:
ok then we also should skip it, it seems to be not part of the spec anyway. @kulmann thoughts?